test/upgrade_test/fission_objects.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+set -e
+
+getVersion() {
+    echo $(git rev-parse HEAD)
+}
+
+getDate() {
+    echo $(date -u +'%Y-%m-%dT%H:%M:%SZ')
+}
+
+getGitCommit() {
+    echo $(git rev-parse HEAD)
+}
+
+generate_test_id() {
+    echo $(((10000 + $RANDOM) % 99999))
+}
+
+dump_system_info() {
+    echo "System Info"
+    go version
+    docker version
+    kubectl version
+    helm version
+}
+
+install_stable_release () {
+    echo "Creating namespace $ns"
+    kubectl create ns $ns
+    echo "Installing Fission $STABLE_VERSION"
```

Comment:
From 1.13 , we need to create CRDs before installing. Maybe we can have an if statement here